### ISSUE TITLE
Add pihole v6 compatibility

### ISSUE THREAD

**Issue by miloit** [tien/pi-hole-connect#196]
Add pihole v6 compatibility

**Comment by amilopowers** [comment]
The API seems to been rewritten completely: https://discourse.pi-hole.net/t/authenticating-to-v6-api/66010

**Comment by timi35** [comment]
Pihole 6 final is now available. Please change your app for the new version.

Thanks.

**Comment by tien** [comment]
@TiredSysOp yes, unfortunately I'm only working on this part-time and won't have time to handle the complexity of supporting both versions. 😢
